app/views/admin/events/show.html.erb

```diff
@@ -69,10 +69,10 @@ as well as a link to its edit page.
 
 <section class="main-content__body">
   <dl>
-    <% page.attributes.each do |attribute| %>
-      <dt class="attribute-label" id="<%= attribute.name %>">
+    <% page.attributes.values[0].each do |attribute| %>
```

Comment:
I don't like that we "magically" access `values[0]` and happen to know that those are the attributes. Maybe we could handle it as it is done in the administrate template?
https://github.com/thoughtbot/administrate/blob/main/app/views/administrate/application/show.html.erb#L54C10-L54C54

```suggestion
    <% page.attributes.each do |_title, attributes| %>
      <% attributes.each do |attribute| %>
```

Reply:
Ah yeah nice I searched not in the template but in the documentation for it. Let me change it.

Reply:
so I corrected that, not really sure if I like that nested each but if they use it that way too it should be fine :/